perun-core/src/main/java/cz/metacentrum/perun/core/entry/SecurityTeamsManagerEntry.java

```diff
@@ -200,17 +200,44 @@ public SecurityTeam getSecurityTeamByName(PerunSession sess, String name) throws
 	}
 
 	@Override
-	public List<User> getAdmins(PerunSession sess, SecurityTeam securityTeam) throws InternalErrorException, PrivilegeException, SecurityTeamNotExistsException {
+	public List<User> getAdmins(PerunSession sess, SecurityTeam securityTeam, boolean onlyDirectAdmins) throws InternalErrorException, PrivilegeException, SecurityTeamNotExistsException {
 		Utils.checkPerunSession(sess);
 		getSecurityTeamsManagerBl().checkSecurityTeamExists(sess, securityTeam);
 
 		if (!AuthzResolver.isAuthorized(sess, Role.SECURITYADMIN, securityTeam)) {
 			throw new PrivilegeException(sess, "getAdmins");
 		}
 
-		return getSecurityTeamsManagerBl().getAdmins(sess, securityTeam);
+		return getSecurityTeamsManagerBl().getAdmins(sess, securityTeam, onlyDirectAdmins);
 	}
 
+        @Deprecated
+        @Override
+        public List<User> getDirectAdmins(PerunSession sess, SecurityTeam securityTeam) throws InternalErrorException, SecurityTeamNotExistsException, PrivilegeException {
```

Comment:
I don't think is good to create "deprecated" method from start. I think that getAdmins with "onlyDirectAdmins" parameter is enough.